src/components/o-testimonials/_template.js

```diff
@@ -0,0 +1,22 @@
+import html from 'nanohtml';
+
+export default ({
+  title,
+  subtitle,
+  autoRotateEnabled = true,
+  autoRotateTime = 5000,
+}, childrenFragment) => html`
+  <article class="o-testimonials__background js-o-testimonials" 
+  data-auto-rotate-enabled="${autoRotateEnabled}" 
+  data-auto-rotate-time="${autoRotateTime}">
+    ${title && html`<h1 class="o-testimonials__title">${title}</h1>`}
+    ${subtitle && html`<p class="o-testimonials__subtitle">${subtitle}</p>`}
+    <div class="o-testimonials__navigator">
+      <div class="o-testimonials__arrow-left js-o-testimonials__control-left"></div>
```

Comment:
why do we need empty divs?

Reply:
@Ben-Zahler 
@nakovskijosif 
I would really like to see an issue describing the features of this component including a design, otherwise it's hard to tell if this is okay or not 🤔

Reply:
I use these 2 divs to draw the left and right arrow with the css below.
border: solid #fff;
border-width: 2px 0 0 2px;
transform: rotate(-45deg);
Do you prefer other way of doing this? If so let me know. I can change this.

Reply:
Okay, so they need to be clickable, than I would suggest `<button>`

Reply:
Done.